GauchoGrub/app/src/main/java/edu/ucsb/cs/cs190i/gauchogrub/gauchogrub/GGApp.java

```diff
@@ -0,0 +1,32 @@
+package edu.ucsb.cs.cs190i.gauchogrub.gauchogrub;
+
+import android.app.Application;
+import edu.ucsb.cs.cs190i.gauchogrub.gauchogrub.db.models.Models;
+
+import io.requery.Persistable;
+import io.requery.android.sqlite.DatabaseSource;
+import io.requery.rx.RxSupport;
+import io.requery.rx.SingleEntityStore;
+import io.requery.sql.Configuration;
+import io.requery.sql.EntityDataStore;
+
+public class GGApp extends Application {
+
+    private SingleEntityStore<Persistable> dataStore;
+
+    /**
+     * @return {@link EntityDataStore} single instance for the application.
+     * <p/>
+     * Note if you're using Dagger you can make this part of your application level module returning
+     * {@code @Provides @Singleton}.
+     */
+    SingleEntityStore<Persistable> getData() {
```

Comment:
@elswenson @andrewtran1995 Here's how this function should be used (within an activity):
`dataStore = ((GGApp) getApplication()).getData();`